bootstrap-app/src/cmr/bootstrap/data/bulk_index.clj

```diff
@@ -148,10 +149,13 @@
                 (name concept-type)
                 (:provider-id provider)))
   (let [db (helper/get-metadata-db-db system)
+        params (merge {:concept-type concept-type 
+                       :provider-id (:provider-id provider)}
+                      (when (cc/generic-concept? concept-type)
+                        {:schema (name concept-type)}))
```

Comment:
Do we need to check that the version is valid here?

Reply:
I don't think so, as it has already been checked by the ingest code when it first got indexed.  This feature is getting the already ingested concepts from the database and just re-indexing them.

Reply:
what happens if the value has been changed in either direction? Should we reindex things which have been turned off?

Reply:
@jceaser that is an interesting question. If we want to be able to reindex things which have been turned off, (for example, someone hits the endpoint /bulk_index/grids , but we have removed grids from the official supported generic docs list) then I believe we will need to add a way to track historical generic documents (that is, a list of all generic document types that have been supported in the past), which I think may warrant a discussion on how to do.

Reply:
@eudoroolivares2016 @eereiter Right now this is working as just "index all grids" or "index all order options" etc without regard to version. I think adding the ability to reindex all grids of a certain version is a neat idea.